src/client.py

```diff
@@ -0,0 +1,40 @@
+import socket
+import sys
+
+infos = socket.getaddrinfo('127.0.0.1', 3000)
+
+stream_info = [i for i in infos if i[1] == socket.SOCK_STREAM][0]
+
+client = socket.socket(*stream_info[:3])
+
+client.connect(stream_info[-1])
+
+message = "test message from client"
```

Comment:
You should be getting the message to send from the command line arguments, it should not be hard coded.